.github/workflows/release.yml

```
@@ -17,25 +17,30 @@ jobs:
     runs-on: ubuntu-latest
     name: "Bump version"
     outputs:
-      version: ${{ steps.cz.outputs.version }}
+      version: ${{ steps.tag.outputs.version }}
     steps:
       - uses: actions/checkout@v4
         with:
           fetch-depth: 0  # Fetch tags, which are required to calculate the new version
           token: "${{ secrets.GITHUB_TOKEN }}"
-      - id: cz
-        name: "Generate Changelog and Tag"
-        uses: commitizen-tools/commitizen-action@master
-        with:
-          github_token: ${{ secrets.GITHUB_TOKEN }}
+      - name: "Install commitizen"
+        run: pip install --user -U commitizen
```

optional: wearing my supply-chain-debugging hat

could we add a line of --version output or something, so we have historical data on what version of `commitizen` is working for us (in case we're debugging a break int he future)

```suggestion
        run: |
          pip install --user -U commitizen &&
            cz version --verbose
```

([looks like it](https://github.com/commitizen-tools/commitizen/blob/43ee8a06b2ff003f45e8975cf9487a802cdf6d89/commitizen/commands/version.py#L31) self-reports its name, so don't need to add our own echo or anything to explain in our own  logs)